pkg/interceptors/logger_test.go

```diff
@@ -0,0 +1,167 @@
+package interceptors
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	goLog "log"
+	"net"
+	"testing"
+
+	grpc_testing "github.com/grpc-ecosystem/go-grpc-middleware/testing"
+	mwitkow_testproto "github.com/grpc-ecosystem/go-grpc-middleware/testing/testproto"
```

Comment:
is it normal in Go that we import dependencies in 3 different naming conventions: camelCase (`goLog`), no word separator (`sdklogger`), and snake_case (`grpc_testing`, `mwitkow_testproto`) in the same file? 🤔

Reply:
@terranisu from the specification perspective there is nothing wrong with it. But it is recommended to use lowercase and without separators (`_`), I've changed that. `grpc_testing` & `mwitkow_testproto` I can't change because it is package names that differ from the directory name, not much I can do about it.